src/resources/i18n/translations/en.json

```diff
@@ -148,8 +148,8 @@
 	"ARIA HINT - show more options for this": "Show more options for this",
 	"ARIA HINT - show more player screen options": "Show more player screen options",
 	"ARIA HINT - show the chapters for this episode": "Show the chapters for this episode",
-	"ARIA HINT - sign up for a premium membership": "Sign up for a premium membership",
-	"ARIA HINT - sign up for your premium membership": "Sign up for your premium membership",
+	"ARIA HINT - sign up for a premium membership": "Sign up for a Premium membership",
```

Comment:
```suggestion
	"ARIA HINT - sign up for a premium membership": "Sign up for a Podverse Premium",
```

Reply:
Can you change this to "Sign up for a Podverse Premium membership"? Saying "Premium" by itself is not as clear that it is a subscription membership.

Reply:
That is the point.